src/drawers/drawer.ts

```diff
@@ -60,6 +74,37 @@ export class Drawer {
   }
 
   /**
+   * Retieves a function that can supply initial settings to entering elements.
+   * this function is typically supplied by the plot using the Drawer
+   */
+  public initializer(): () => AttributeToProjector;
+  /**
+   * Sets the initializer function for the Drawer.
+   * This function returns an AttributeToProjector that is applied
+   * to the new elements appended to the enter() selection
+   * Typically set from _createDrawer in the plot
+   *
+   * @param {() => AttributeToProjector} the function.
+   * @returns {Drawer} The calling Drawer.
+   */
+  public initializer(fnattrToProjector: () => AttributeToProjector): Drawer;
+  public initializer(fnattrToProjector?: () => AttributeToProjector): any {
+    if (fnattrToProjector == null) {
+      return this._initializer;
+    }
+    this._initializer = fnattrToProjector;
+    return this;
+  }
```

Comment:
Mainly curious: why are we setting up the initial state here instead of using a null animator to set up the initial state at the plot level?